greedy_methods/greedy_coin_change.py

```diff
@@ -0,0 +1,69 @@
+"""
+This is a pure Python implementation of the greedy coin change algorithm.
+
+For more information on the coin change problem see:
+https://en.wikipedia.org/wiki/Change-making_problem
+
+This algorithm computes the needed coins from a given set of coins that
+make up a given amount.
+It works greedy, so it chooses the biggest possible coin that fits
+in the remaining amount until
+the remaining sum is <=0.
+
+As seen in the third test, the greedy algorithm is not always optimal.#
+For more information:
+https://stackoverflow.com/questions/13557979/why-does-the-greedy-coin-change\
+-algorithm-not-work-for-some-coin-sets
+"""
+
+
+def compute_coins(amount: int, coin_set: set[int]) -> list[int]:
+    """
+    Implementation of the greedy coin change algorithm.
+    :param amount: Amount to compute the coin change for
```

Comment:
Please keep an empty line after the summary.

```suggestion
    Implementation of the greedy coin change algorithm.
    
    :param amount: Amount to compute the coin change for
```